crates/telio-model/src/features.rs

```diff
@@ -518,13 +536,15 @@ impl Default for FeaturePmtuDiscovery {
     }
 }
 
-#[derive(Clone, Debug, Default, PartialEq, Eq, Deserialize)]
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, SmartDefault)]
 /// Encompasses all of the possible features that can be enabled
 pub struct Features {
     /// Additional wireguard configuration
     #[serde(default, deserialize_with = "FeatureWireguard::default_on_null")]
     pub wireguard: FeatureWireguard,
     /// Nurse features that can be configured for QoS
+    #[serde(default = "Features::default_nurse")]
+    #[default(Features::default_nurse())]
```

Comment:
I would be okay with that if we could get rid of the function call from `#[serde(default = "Features::default_nurse")]` but since we can't, I think this is a cleaner solution